f8a_worker/workers/stackaggregator_v2.py

```diff
@@ -13,6 +13,7 @@
 from f8a_worker.graphutils import GREMLIN_SERVER_URL_REST, LICENSE_SCORING_URL_REST
 from f8a_worker.utils import get_session_retry
 
+import logging
 
 def extract_component_details(component):
```

Comment:
Is this function called from somewhere else? It would be more suitable to place it to correspondent class (`StackAggregatorV2Task` I believe) to encapsulate logic.